components/permit-holders/PersonalInformationCard.tsx

```diff
@@ -40,12 +42,13 @@ export default function PersonalInformationCard(props: PersonalInformationProps)
       <VStack spacing="12px" align="left">
         <Box>
           <Text as="p" textStyle="body-regular">
-            Date of Birth: {applicant.dateOfBirth}
+            Date of Birth:{' '}
+            {applicant?.dateOfBirth && new Date(applicant.dateOfBirth).toLocaleDateString('en-CA')}
```

Comment:
```suggestion
            {new Date(applicant.dateOfBirth).toLocaleDateString('en-CA')}
```
`applicant.dateOfBirth` is non-null